mailer.rst

```diff
@@ -785,6 +1016,41 @@ you have a transport called ``async``, you can route the message there:
             ],
         ]);
 
+    .. code-block:: php-standalone
```

Comment:
I'm not very sure about this code block as well. Seems very advanced to set-up message bus + mailer in a standalone application.